README.md

```diff
@@ -165,3 +165,38 @@ function rex_url_shortener(rex_extension_point $ep) {
 	return $url;
 }
 ```
+
+## SEO-Methoden
+
+Für die ordnungsgemäße Ausgabe müssen die YRewrite-Tags für Titel, Beschreibung u.a. durch die SEO-Klasse des URL-Addons ausgetauscht werden:
+
+```php
+$urlSeo = new Url\Seo();
+    echo $urlSeo->getTitleTag().PHP_EOL;
+    echo $urlSeo->getDescriptionTag().PHP_EOL;
+    echo $urlSeo->getRobotsTag().PHP_EOL;
+    echo $urlSeo->getHreflangTags().PHP_EOL;
+    echo $urlSeo->getCanonicalUrlTag().PHP_EOL;
+```
```

Comment:
Wie wäre es hiermit?

**Bisher**
```php
$seo = new rex_yrewrite_seo();
echo $seo->getTitleTag();
echo $seo->getDescriptionTag();
echo $seo->getRobotsTag();
echo $seo->getHreflangTags();
echo $seo->getCanonicalUrlTag();
```

**URL-Adddon**
```php
$seo = new Url\Seo();
// Rest unverändert
```

Damit können wir hervorheben, dass nur die Class und keine Methoden ausgetauscht werden muss.

Reply:
ich habe deinen Vorschlag leicht verändert.

Reply:
Ich hatte den Part direkt von yrewrite genommen und würde gern eine neue Variable hier vermeiden. Hier steht zwar alles hübsch untereinander, in Templates habe ich aber bei der Usern schon andere Varianten gesehen, sodass man leicht eine Anpassung übersehen kann und in einen Whoops läuft.